### DISCUSSION TITLE
UI scaling

### DISCUSSION THREAD

**Discussion by ghost** [excalidraw/excalidraw#8816] [General]
I would like to have different scaling for excalidraw ui like idk maybe it happened after some update but now excalidraw ui is too big it auto scales with my obsidian ui but the thing is i have bad eyes and i need big obsidian ui so i can read but the excalidraw ui is too big and covers large area. I think we should have a feature to have separate scaling for excalidraw ui elements.

**Comment by Mrazator** [comment]
Makes sense, let's track it here https://github.com/excalidraw/excalidraw/issues/8839
